shared/src/commonMain/kotlin/org/hyperskill/app/problems_limit/presentation/ProblemsLimitFeature.kt

```diff
@@ -1,23 +1,28 @@
 package org.hyperskill.app.problems_limit.presentation
 
+import org.hyperskill.app.problems_limit.domain.model.ProblemsLimitScreen
 import org.hyperskill.app.subscriptions.domain.model.Subscription
 import kotlin.time.Duration
 
 object ProblemsLimitFeature {
-    internal sealed interface State {
+    sealed interface State {
```

Comment:
Yes, it's impossible, because this state is used in HomeFeature, we don't have view state there so it forces us to make problems limit feature state public. We can leave an `internal` modifier, but the we need to add ViewState to home feature it's out of this task

Reply:
> Yes, it's impossible, because this state is used in HomeFeature, we don't have view state there so it forces us to make problems limit feature state public.

OK